ros/src/twist_controller/dbw_node.py

```diff
@@ -56,49 +69,135 @@ def __init__(self):
         self.brake_pub = rospy.Publisher('/vehicle/brake_cmd',
                                          BrakeCmd, queue_size=1)
 
-        # TODO: Create `TwistController` object
-        # self.controller = TwistController(<Arguments you wish to provide>)
-
-        # TODO: Subscribe to all the topics you need to
-        rospy.Subscriber('/vehicle/dbw_enabled', Bool, self.dbw_enabled_cb)
+        self.controller = Controller(vehicle_mass,
+                                     fuel_capacity,
+                                     brake_deadband,
+                                     decel_limit,
+                                     accel_limit,
+                                     wheel_radius,
+                                     wheel_base,
+                                     steer_ratio,
+                                     max_lat_accel,
+                                     max_steer_angle)
+
+        rospy.Subscriber('/current_velocity',
+                         TwistStamped,
+                         self.current_velocity_cb)
```

Comment:
do you think it will be better if we set queue_sizes to 1 for all subsrcibers to avoid backlog of stale messages?

Reply:
yes... good call. I'll set the queue_sizes to 1.